unabridged-works-volume-1/original/05-babylon-the-great.adoc

```diff
@@ -205,7 +205,7 @@ and feel in yourselves what it is which is to be led out, and what is to lead yo
 that your feet may be guided to, and set firm upon, Mount Zion;
 where the life rules over all her enemies.
 
-BABYLON is the spiritual fabric of iniquity;
```

Comment:
I wasn't sure if this was capitalized for emphasis or not.

Reply:
I think you did right in lowercasing it, it seems sort of like the first word after the prefatory stuff, so probably was capitalized for stylistic reasons.